java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -272,6 +272,32 @@ CompletableFuture<Double> zaddIncr(
      */
     CompletableFuture<Map<String, Double>> zpopmax(String key);
 
+    /**
+     * Blocks the connection until removes and returns a member with the highest score from the sorted
+     * sets stored at the specified <code>keys</code>.<br>
+     * <code>BZPOPMAX</code> is the blocking variant of {@link #zpopmax(String)}.<br>
+     * A member with the highest score is popped from first sorted set that is non-empty, with the
+     * given <code>keys</code> being checked in the order that they are given.
+     *
+     * @see <a href="https://redis.io/commands/bzpopmax/">redis.io</a> for more details.
+     * @apiNote <code>BZPOPMAX</code> is a client blocking command, see <a
+     *     href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands">Blocking
+     *     Commands</a> for more details and best practices.
+     * @param keys The keys of the sorted sets.
+     * @param timeout The number of seconds to wait for a blocking <code>BZPOPMAX</code> operation to
+     *     complete. A value of <code>0</code> will block indefinitely.
+     * @return An array containing the key where element was popped out (as <code>String</code>), the
+     *     element itself (as <code>String</code>) and the element score (as <code>Double</code>).<br>
+     *     The data could be interpreted in format as <code>[key, element, score]</code>.<br>
```

Comment:
Incorprate this sentence to the one above like to be less wordy:
an array containing the key name, the popped element and the element score. [key, element, score]